snap/snapcraft.yaml

```diff
@@ -25,6 +25,7 @@ apps:
     restart-delay: 2s
     plugs:
       - snap-themes-control
+      - login-session-observe
```

Comment:
We're going to need to request autoconnect of this interface, which is fine.

Reply:
Anyway, the code is safe and still works if it isn't connected (although it will spit errors each two seconds, just like now).